css/properties/align-items.json

```diff
@@ -8,10 +8,19 @@
             "webview_android": {
               "version_added": "4.4"
             },
```

Comment:
`webview_android` should be the same as `chrome` and be the last in the `support` block (see issue #398 and PR #1882)

Reply:
I agree that it should be in alphabetical order. But I'm not convinced that `webview_android` should be the same as `chrome`? According to [caniuse](https://caniuse.com/#feat=flexbox) the full spec was implemented in `4.4`.

Reply:
Or did you mean the same as `chrome` but with `"version_added": "4.4"` instead of `"version_added": "29"`?

Reply:
Chrome Developer Relations here. He means that Android webview follows Chrome version numbering, not Android version numbering.

Reply:
Oh, I see. Didn't know. Thanks